README.md

```diff
@@ -16,7 +16,7 @@
 
 Quirrel makes job queueing as easy as pie. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
 
-Quirrel values ...
+Quirrel values...
```

Comment:
According to https://www.grammarly.com/blog/ellipsis/, an ellipsis should be preceded by a space.
```suggestion
Quirrel values ...
```

Reply:
You learn something new each day!